### ISSUE TITLE
View PRs at the orignization level rather than repo level

### ISSUE THREAD

**Issue by Wolftousen** [microsoft/vscode-pull-request-github#1458]
This may be my lack of knowledge as i've only used this plugin for a few days now, but it would be nice to have an option to have the queries run at the organization level instead of the individual repo level.

Our organization uses a multirepo set up and we generally have 10+ repos open in vscode at any given time.  While this isn't so much an issue for the average dev, being the architect and primary code reviewer, it gets cumbersome having to click on a repo, check for PRs, and repeat.  I'd like to just have queries that show me all PRs for all Repos/Branches in the extension view.

If this is already possible and i'm just missing something awesome, can you enlighten me?

If not, it would be great to see this added as an option.

**Comment by RMacfarlane** [comment]
@Wolftousen Thanks for the write up! This is currently not possible, but is certainly a reasonable feature request.

**Comment by vstrimaitis** [comment]
Yeah, this feature would be amazing to have in the extension! Similar to what @Wolftousen mentioned, I also have about 40 repositories active in my workspace at any given time, so being able to check at a glance where my review is needed would drastically improve my workflow.

**Comment by alexr00** [comment]
We have this now!
```
"githubPullRequests.queries": [
    {
      "label": "Local Pull Request Branches",
      "query": "default"
    },
    {
      "label": "My Microsoft",
      "query": "org:Microsoft is:open assignee:${user}" // specify an org like this
    },
    {
      "label": "All Open",
      "query": "default"
    }
  ],
```
